src/server/journalism.js

```diff
@@ -48,15 +44,9 @@ export const journalismTeam = (res, team) => {
             ));
           }
         )
-      ).then((data) => (
-        { ...all, matches: data }
-      ));
-    }).then((all) => (
-      { matches: all.matches, dbInfo: all.dbInfo, team  }
-    )).then((all) =>
-      res.end(JSON.stringify(all))
-    );
-};
+      ).then((data) => ({ ...all, matches: data }))
+    ).then((all) => ({ matches: all.matches, team }))
+    .then((all) => res.end(JSON.stringify(all)));
```

Comment:
For the record, this was indented how it was specifically so the .then(s all lined up, not that it matters much